css/styles.css

```diff
@@ -243,16 +247,26 @@ select {
   align-items: center;
   text-align: center;
   color: #fff;
-  right: 10px;
-  margin-top: 30px;
-  padding: 2.5rem;
+  margin-top: 10px;
+  padding: 1.5rem;
+  left: 0;
+}
+
+.contact-btn {
+  background-color: #f6f6fa;
+  color: rgb(11, 159, 239);
```

Comment:
- [OPTIONAL] The color of the contact button text is slightly different from the template. It would be nice if you use the same color to match the whole portfolio color pallette.